tableaudocumentapi/connection.py

```diff
@@ -92,3 +97,33 @@ def username(self, value):
         """
         self._username = value
         self._connectionXML.set('username', value)
+
+    ###########
+    # authentication
+    ###########
+    @property
+    def authentication(self):
+        return self._authentication
+
+    ###########
+    # authentication
+    ###########
+    @authentication.setter
+    def authentication(self, value):
+        self._authentication = value
+        self._connectionXML.set('authentication', value)
+
+    ###########
+    # dbclass
+    ###########
+    @property
+    def dbclass(self):
+        return self._class
+
+    ###########
+    # dbclass
+    ###########
+    @dbclass.setter
```

Comment:
We could just leave this as 'read-only' since changing the class might have other consequences I'm not familiar with.